general/clients/css-customization.md

```diff
@@ -5,7 +5,7 @@ title: CSS Customization
 
 # CSS Customization
 
-In ` Dashboard > General `, the "Custom CSS" field can be used to override current CSS in Jellyfin's stylesheet.
+In `Dashboard > General`, the Custom CSS field can be used to override current CSS in Jellyfin's stylesheet.
```

Comment:
I think field names etc. should be quoted.

Reply:
done